File: C++/basic/FindElemInArray.cpp
int findElem(int arr[], int k){
	int len = sizeof(arr)/sizeof(arr[0]);
    if(len  < 1) return -1;
    for(int i = 0 ; i < len ; i++){
        if(arr[i] == k) return i;
    }
    return -1;
}

File: Python/basic/FindElemInArray.py
def findElem(arr, k) : 
    for i in range(len(arr)) : 
        if(arr[i] == k) : return i;

    return -1;

arr = [1, 2, 3, 4]
print(findElem(arr, 3))   
print(findElem(arr, 5)) 
